src/pages/token/RewardsPage.tsx

```diff
@@ -54,14 +55,13 @@ const RewardsPage = () => {
   const { usdcDenom } = useTokenConfigs();
   const usdcDecimals = 24; // hardcoded solution; fix in OTE-390
 
-  const { totalRewards } = useAppSelector(getStakingRewards, shallowEqual) ?? {};
+  const stakingRewards = BonsaiHooks.useStakingRewards().data;
```

Comment:
mega-nit: could shorten to orEmptyObj(BonsaiHooks.useStakingRewards().data?.totalRewards) instead of destructuring